packages/amplify_datastore/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.0-next.0+1
+
+- Minor bug fixes and improvements
```

Comment:
*nit:* I had a similar comment the automated version bump PR. In this situation, "Internal dependencies bumped to latest version" would make more sense to me. "Minor bug fixes and improvements" isn't necessarily incorrect, but since the changes were not in this package it doesn't seems like the most appropriate message.

Reply:
Hmm, not sure it's worth the effort distinguishing these scenarios. This message appears in one of three cases:
- The DataStore package's list of commits consisted purely of chores
- The DataStore package's list of commits consisted purely of cascading updates in dependencies
- Some mix of the two

It would be probably more effort than it's worth to suss out which of the three actually occurred. "Minor bug fixes and improvements" seems to cast a wider net and is applicable in all three scenarios. In this case, the DataStore bump is due to a bump in its dependencies plus a change in the README, so both messages would really be needed to describe the full range of changes.

Maybe instead of an either/or, we could add "Internal dependencies bumped to latest version" in the case of cascading updates. This could be done with some sort of marker commit.